packages/ui/src/1_atoms/Button/Button.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import React from 'react';
+
+import { Button } from './Button';
+// import { ButtonSize, ButtonStyle } from './Button.types';
+
+describe('Button', () => {
+  it('renders a button', () => {
+    render(<Button text="Button" />);
+    expect(screen.getByText('Button')).toBeInTheDocument();
+  });
+
+  it('eventHandler called on click', () => {
+    const handleClick = jest.fn();
+    render(<Button text="Button" onClick={handleClick} />);
+    const button = screen.getByText('Button');
+    fireEvent.click(button);
```

Comment:
Use `userEvent` instead of `fireEvent` as the former is performing actions in more realistic (from a user perspective) way. See https://testing-library.com/docs/user-event/intro#differences-from-fireevent for more details.